plugins/movetoarchive.koplugin/main.lua

```diff
@@ -50,7 +50,7 @@ function MoveToArchive:addToMainMenu(menu_items)
             {
                 text = _("Go to archive folder"),
                 callback = function()
-                    if self.archive_dir_path then
+                    if self.archive_dir_path and util.directoryExists(self.archive_dir_path) then
```

Comment:
Is it a problem if you pass nil? Otherwise that's a bit redundant. (Not a request for change.)

Reply:
If the argument is nil, I think it will just ask to set a folder right a way?

Reply:
Ah right, sorry, that was in fact under the diff too.